backend/src/main/kotlin/org/loculus/backend/service/groupmanagement/GroupManagementDatabaseService.kt

```diff
@@ -88,6 +88,15 @@ class GroupManagementDatabaseService(
         )
     }
 
+    fun updateGroup(groupId: Int, group: NewGroup, authenticatedUser: AuthenticatedUser) {
+        groupManagementPreconditionValidator.validateThatUserExists(authenticatedUser.username)
+
+        groupManagementPreconditionValidator.validateUserIsAllowedToModifyGroup(groupId, authenticatedUser)
+
+        // TODO do the editing
+        // Update the groupId group with the info in NewGroup
```

Comment:
Btw Kotlin has a `TODO()` function that usually makes your code compile, which is quite nice. (Not necessary here, because you don't need to return anything)

That reminds me of: should the new endpoint maybe return the new (updated) group information?

Reply:
The point of returning the new information would be as a sanity check or would it serve any other purpose?

Reply:
Yes, sanity check / convenience for users and easier testing

Reply:
thanks for the tip regarding TODO! I've also updated the function to return the group now 9ec8bda